articles/active-directory-b2c/conditional-access-user-flow.md

```diff
@@ -157,6 +157,123 @@ After you've added the Azure AD Conditional Access policy, enable conditional ac
+| Admin confirmed user compromised | Offline | This detection indicates an admin has selected 'Confirm user compromised' in the Risky users UI or using riskyUsers API. To see which admin has confirmed this user compromised, check the user's risk history (via UI or API). |
+| Malicious IP address | Offline | This detection indicates sign-in from a malicious IP address. An IP address is considered malicious based on high failure rates because of invalid credentials received from the IP address or other IP reputation sources. |
+| Password spray | Offline | A password spray attack is where multiple usernames are attacked using common passwords in a unified brute force manner to gain unauthorized access. This risk detection is triggered when a password spray attack has been performed. |
+
+Organizations should choose one of the following options to enable a sign-in risk-based Conditional Access policy requiring multi-factor authentication (MFA) when sign-in risk is medium OR high.
+
+### Enable with Conditional Access policy
+
+1. Sign in to the **Azure portal**.
+1. Browse to **Azure AD B2C** > **Security** > **Conditional Access**.
+1. Select **New policy**.
+1. Give your policy a name. We recommend that organizations create a meaningful standard for the names of their policies.
+1. Under **Assignments**, select **Users and groups**.
+   1. Under **Include**, select **All users**.
+   1. Under **Exclude**, select **Users and groups** and choose your organization's emergency access or break-glass accounts. 
+   1. Select **Done**.
+1. Under **Cloud apps or actions** > **Include**, select **All cloud apps**.
+1. Under **Conditions** > **Sign-in risk**, set **Configure** to **Yes**. Under **Select the sign-in risk level this policy will apply to** 
+   1. Select **High** and **Medium**.
+   1. Select **Done**.
+1. Under **Access controls** > **Grant**, select **Grant access**, **Require multi-factor authentication**, and select **Select**.
+1. Confirm your settings and set **Enable policy** to **On**.
+1. Select **Create** to create to enable your policy.
+
+### Enable with Conditional Access APIs
+
+The steps to create a Sign-in risk-based Conditional Access policy with Conditional Access APIs is documented in the article, [Conditional Access APIs: Sign-in risk-based Conditional Access](https://docs.microsoft.com/azure/active-directory/conditional-access/howto-conditional-access-apis). We will use that document as a reference to create a policy called "Template 1: Require MFA for medium + sign-in risk" using the APIs.
+
```

Comment:
Nit, since this link doesn't go directly to the sign-in risk sample

```suggestion
The steps to create a Sign-in risk-based Conditional Access policy with Conditional Access APIs is documented in a sample here, [Conditional Access APIs](https://docs.microsoft.com/en-us/azure/active-directory/conditional-access/howto-conditional-access-apis#graph-api). We will use the sample as a reference to create a policy called "Template 1: Require MFA for medium + sign-in risk" using the APIs.

```